ci/environment-win.yml

```diff
@@ -20,3 +20,5 @@ dependencies:
   - python-libarchive-c
   - numpy
   - nomkl
+  - diskcache
+  - appdirs
```

Comment:
I recently discovered that the successor library to `appdirs` is [`platformdirs`](https://pypi.org/project/platformdirs/).